### PR TITLE
add ci test for bulk load

### PR DESCRIPTION
**Pull request by marsishandsome**
Signed-off-by: marsishandsome <marsishandsome@gmail.com>

close https://github.com/tikv/migration/issues/6

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          repository: tikv/client-java
+          path: client-java
+
+      - name: Build Client Java
+        run: mvn --file client-java/pom.xml clean install -DskipTests
+
+      - name: Run TiKV Cluster
+        run: /home/runner/.tiup/bin/tiup playground nightly --mode tikv-slim --kv 3 --monitor=false &
+
+      - name: Wait TiKV Cluster
```
Comment (tisonkun) [inline .github/workflows/test.yml:52]: can we wait TiKV cluster online determinately?

**Review by tisonkun — COMMENTED** [review]
Generally looks good. One comment inline.

File: .github/workflows/test.yml (hunk 1 above)
Comment (marsishandsome) [reply .github/workflows/test.yml]: yes. we can do it in this issue https://github.com/tikv/migration/issues/19

Comment (tisonkun) [reply .github/workflows/test.yml]: Do you mean we do it as a follow up? If so, this PR is ok to merge for me.

**Review by tisonkun — APPROVED** [review]

**Pull request merged**